src/Libraries/CoreNodeModels/String.cs

```diff
@@ -59,13 +62,50 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
         }
     }
 
+    [NodeName("String from Object And Format")]
```

Comment:
How about renaming this to `Formatted String from Object`?

Reply:
I don't hate it!